optimize/self-managed/optimize-deployment/migration-update/camunda-8/instructions.md

```diff
@@ -0,0 +1,118 @@
+4. Stop the old ES instance.
+5. Start the new ES instance and check that everything looks fine.
+
+Although the steps above summarize the basic update procedure, we recommend reading through the Elasticsearch documentation to avoid any potential issues.
+
+:::note Heads Up!
+
+Note that the following updates are not supported by Elasticsearch:
+
+- 6.8 to 7.0.
+- 6.7 to 7.1.–7.X (where X>1, e.g. 7.5)
+
+:::
+
+### 3. Perform the migration
+
+The update can also be executed as part of the Optimize startup. In order to make use of this functionality, the command flag `--upgrade` has to be passed to the Optimize startup script.
+
+In Docker environments this can be achieved by overwriting the default command of the docker container (being `./optimize.sh`), e.g. like in the following [docker-compose](https://docs.docker.com/compose/) snippet:
+
+```
+version: '2.4'
+
+services:
+  optimize:
+    image: registry.camunda.cloud/optimize-ee/optimize:8-latest
+    command: ["./optimize.sh", "--upgrade"]
+```
+
+However, as this may prolong the container boot time significantly which may conflict with [container status probes](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/) in managed environments like [Kubernetes](https://kubernetes.io/) we recommend using the [init container](https://kubernetes.io/docs/concepts/workloads/pods/init-containers/) feature there to run the update:
+
+```
+  labels:
+    app: optimize
+spec:
+   initContainers:
+     - name: migration
+       image: registry.camunda.cloud/optimize-ee/optimize:8-latest
+       command: ['./upgrade/upgrade.sh', '--skip-warning']
+   containers:
+     - name: optimize
+       image: registry.camunda.cloud/optimize-ee/optimize:8-latest
+```
+
+### 4. Resume a canceled update
+
+Updates are resumable. So if the update process got interrupted either manually or due to an error you don't have to restore the Elasticsearch backup and start over but can simply rerun the update. On resume previously completed update steps will be detected and logged as being skipped. In the following log example **Step 1** was previously completed and is thus skipped:
+
+```
+./upgrade/upgrade.sh
+...
+INFO UpgradeProcedure - Skipping Step 1/2: UpdateIndexStep on index: process-instance as it was found to be previously completed already at: 2020-11-30T16:16:12.358Z.
+INFO UpgradeProcedure - Starting step 2/2: UpdateIndexStep on index: decision-instance
+...
+```
+
+### 5. Typical errors
+
+- Using an update script that does not match your version:
+
+```bash
+Schema version saved in Metadata does not match required [2.X.0]
+```
+
+Let's assume have Optimize 2.1 and want to update to 2.3 and use the jar to update from 2.2 to 2.3. This error occurs because the jar expects Elasticsearch to have the schema version 2.1. This is because you downloaded the wrong Optimize artifact which contained the wrong update jar version.
```

Comment:
```suggestion
Let's assume have Optimize 8.4 and want to update to 8.6 and use the jar to update from 8.5 to 8.6. This error occurs because the jar expects Elasticsearch to have the schema version 8.4. This is because you downloaded the wrong Optimize artifact which contained the wrong update jar version.
```